components/single_image_view/single_image_view.jsx

```diff
@@ -130,21 +130,13 @@ export default class SingleImageView extends React.PureComponent {
         let fadeInClass = '';
 
         const fileType = getFileType(fileInfo.extension);
-        let styleIfSvgWithDimentions = {};
-        let imageContainerStyle = {};
+        let styleIfSvgWithDimensions = {};
         let svgClass = '';
         if (fileType === FileTypes.SVG) {
             svgClass = 'svg';
-            if (this.state.dimensions.height) {
-                styleIfSvgWithDimentions = {
-                    width: '100%',
-                };
-            } else {
-                imageContainerStyle = {
-                    height: 150,
-                    width: 'auto',
-                };
-            }
+            styleIfSvgWithDimensions = {
```

Comment:
Is this possible to fix with fixed dimensions here in case dimensions are not present? 

This could cause scroll pops with svgs added before the support for svgs dimension calculation on server was added.

Reply:
I suppose so - but I’m not sure what the fixed dimensions should be. You could set a fixed vertical dimension, and maybe at max-width of 100%? I’m trying to think how you constrain with a fixed dimension that won’t bleed over. The last issue was that a fixed height on the container wouldn’t contain the image as it’s inherent height with auto width would cause it to break out.

Reply:
Is there some way I can test the scenario above @sudheerDev ?

Reply:
Ignore my comment of old svg's. 
This would still cause scroll pops with svg's without dimensions because it will change scroll position as it loads. 

Hence the fixed dimensions. Can we not keep height fixed and adjust the width accordingly? Override the image height of inherit?

Reply:
I think we can set a fixed height but I need to understand the image wrapper better. Seems the expanded preview image should be 300px tall. But what happens if you have a portrait oriented image (width wider than height)? I can’t distort the image or crop the edges- well, unless the image wrapper will hide the overflow. I’ll do some experimentation.

Reply:
> Seems the expanded preview image should be 300px tall. But what happens if you have a portrait oriented image (width wider than height)

I would say we don't fixed width set the width. We would have to set `max-width` for which if the image aspect ratio is greater than that yes, it will distort but IMO that is a extremely niche case to solve.

Reply:
Fixing the width and not the height would yield something like this, but only in situations where the dimensions aren't known: ![qiuzuxYERK](https://user-images.githubusercontent.com/44858354/62909731-af629a00-bd32-11e9-99eb-76103cb2265a.gif)

Reply:
I've submitted the CSS changes as per the above, however, I wonder if we shouldn't set a shorter fixed height on the mobile breakpoints.